src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,73 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getUser } from '../../api/api';
+
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: null,
+  };
+
+  componentDidMount() {
+    this.selectedUser();
+  }
+  
+  componentDidUpdate(prevProps) {
+    if (prevProps.userId === this.props.userId) {
+      return;
+    }
+    this.selectedUser();
```

Comment:
```suggestion
    this.selectUser();
```
It would be a better naming, because the name of the function should describe some action